### PR TITLE
poop more

### PR DESCRIPTION
**Pull request by fene**
exports an overloaded function called with a `string` and how many `times` you want to add poop. for incredibly shitty days

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,3 +12,11 @@ module.exports = function (string) {
   var poop = emoji.get('hankey')
   return poop + ' ' + string + ' ' + poop
 }
+
+module.exports = function pooptimes (string, times) {
```
Comment (lfairy) [inline index.js]: Does it make sense to assign to `module.exports` twice in this way?

Comment (fene) [inline index.js]: i mean, we could add it as a separate export. but this way we have `poopthis(string)` and `poopthis(string, amount)`

**Comment by Charlotteis** [comment]
Thanks!!

**Comment by BPScott** [comment]
Poop pedantry: `poopthis(s)` and `poopthis(s, 1)` are equivalent so could you get away with one function with a [default argument](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Default_parameters)?

**Pull request merged**